source/routing/raptor.cpp

```diff
@@ -286,6 +288,9 @@ RAPTOR::compute_all(const vec_stop_point_duration& departures_,
     }
     BOOST_ASSERT( departures.size() > 0 );    //Assert that reversal search was symetric
 
+    if (nb_sol_reader < result.size()) {
+        std::cout << "***** error reader fail" << nb_sol_reader << " " << result.size() << std::endl;
```

Comment:
That's pure debugging for the moment as the reader just do nothing more than printing things.